src/modules/fw_path_navigation/FixedwingPositionControl.cpp

```diff
@@ -1150,7 +1147,10 @@ FixedwingPositionControl::control_auto_velocity(const float control_interval, co
 	Vector2f target_velocity{pos_sp_curr.vx, pos_sp_curr.vy};
 	_target_bearing = wrap_pi(atan2f(target_velocity(1), target_velocity(0)));
 
-	float target_airspeed = adapt_airspeed_setpoint(control_interval, pos_sp_curr.cruising_speed,
+	float adapted_target_airspeed = target_velocity.norm() >  _param_fw_airspd_min.get() ? target_velocity.norm() :
+					_param_fw_airspd_min.get();
```

Comment:
is this necessary? `adapt_airspeed_setpoint()` constrains to min/max. maybe instead simply sanity checking the values (since they come from offboard likely) would be a good idea here (`ISFINITE()` vector form)

Reply:
Sounds good! Would you have opinions of using groundspeed vs airspeed? The interface is exposed as a groundspeed, so using groundspeed would be a more "correct" implementation, while not optimal for fixedwings